.github/dependabot.yml

```diff
@@ -1,11 +1,25 @@
 version: 2
 updates:
-  - package-ecosystem: "github-actions"
-    directory: "/"
+  - package-ecosystem: github-actions
+    directory: /
+    groups:
+      github-actions:
+        patterns:
+          - "*"
+        update-types:
+          - minor
+          - patch
```

Comment:
Why not include major version updates here and below? Shouldn't the PR fire so we're aware the update exists even if we don't want to take it immediately?